com.unity.shadergraph/Editor/Data/SubGraph/SubGraphAsset.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
is Linq still being used in this file?

Reply:
Oh, no-- I'll remove that. Whoops. I forgot how much VS likes to automatically inject imports for C#.